treeherder/model/migrations/0001_initial.py

```diff
@@ -1,533 +1,323 @@
+                ('description', models.TextField(default='fill me', blank=True)),
+                ('active_status', models.CharField(default='active', max_length=7L, blank=True)),
+            ],
+            options={
+                'db_table': 'repository',
+            },
+            bases=(models.Model,),
+        ),
+        migrations.CreateModel(
+            name='RepositoryGroup',
+            fields=[
+                ('id', models.AutoField(serialize=False, primary_key=True)),
+                ('name', models.CharField(max_length=50L)),
+                ('description', models.TextField(default='fill me', blank=True)),
+                ('active_status', models.CharField(default='active', max_length=7L, blank=True)),
+            ],
+            options={
+                'db_table': 'repository_group',
+            },
+            bases=(models.Model,),
+        ),
+        migrations.CreateModel(
+            name='RepositoryVersion',
+            fields=[
+                ('id', models.AutoField(serialize=False, primary_key=True)),
+                ('version', models.CharField(max_length=50L)),
+                ('version_timestamp', models.IntegerField()),
+                ('active_status', models.CharField(default='active', max_length=7L, blank=True)),
+                ('repository', models.ForeignKey(to='model.Repository')),
+            ],
+            options={
+                'db_table': 'repository_version',
+            },
+            bases=(models.Model,),
+        ),
+        migrations.CreateModel(
+            name='UserExclusionProfile',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('is_default', models.BooleanField(default=True)),
+                ('exclusion_profile', models.ForeignKey(blank=True, to='model.ExclusionProfile', null=True)),
+                ('user', models.ForeignKey(related_name='exclusion_profiles', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'db_table': 'user_exclusion_profile',
+            },
+            bases=(models.Model,),
+        ),
+        migrations.AddField(
+            model_name='repository',
+            name='repository_group',
+            field=models.ForeignKey(to='model.RepositoryGroup'),
+            preserve_default=True,
+        ),
+        migrations.AlterUniqueTogether(
+            name='optioncollection',
+            unique_together=set([('option_collection_hash', 'option')]),
+        ),
+        migrations.AddField(
+            model_name='exclusionprofile',
+            name='exclusions',
+            field=models.ManyToManyField(related_name='profiles', to='model.JobExclusion'),
+            preserve_default=True,
+        ),
+        migrations.AlterUniqueTogether(
+            name='datasource',
+            unique_together=set([('project', 'dataset', 'contenttype'), ('host', 'name')]),
+        ),
+    ]
```

Comment:
> This file is auto-generated, so it would be hard to annotate this.

Ah I didn't realise - I'm still not very familiar with Django - reading up more about it is on my neverending list of things to find time for sometime :-)